open-api/rest-catalog-open-api.yaml

```diff
@@ -3804,6 +4110,41 @@ components:
             }
           }
 
+    #  Note that this is a representative example response for use as a shorthand in the spec.
+    #  The fields `message` and `type` as indicated here are not presently prescriptive.
+    MisdirectedRequestResponse:
+      description:
+        Misdirected Request. The request was directed to a server that is not able to produce a response.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/IcebergErrorResponse'
+          example: {
+            "error": {
+              "message": "Fail to plan table scan: must perform preplan before planning a table scan",
+              "type": "PreplanTableRequiredException",
+              "code": 421
+            }
+          }
+
+    #  Note that this is a representative example response for use as a shorthand in the spec.
+    #  The fields `message` and `type` as indicated here are not presently prescriptive.
+    UnprocessableContentResponse:
```

Comment:
I think @rdblue you suggested using 413 for payload too big, but based on the HTTP code, it looks like 413 is for Request payload too big, but what we want here is response payload too big. I think it can be covered by 422, so I changed to that with an example

Reply:
+1